src/manifest.json

```diff
@@ -1,22 +1,22 @@
 {
-    "manifest_version": 2,
+    "manifest_version": 3,
     "name": "CatchAll Bird",
     "description": "Enable Thunderbird to handle CatchAll email addresses.",
-    "version": "1.1.2",
+    "version": "1.2.0",
     "author": "Jabb0",
     "browser_specific_settings": {
         "gecko": {
             "id": "catchallbird@jabb0",
-            "strict_min_version": "78.0"
+            "strict_min_version": "128.0"
```

Comment:
Did you find something that states 128 does not support manifest 2 at all?  

I wonder if we can keep manifest v2, and do only the `.id` and `delete encryption` change.

However, as I think everbody should upgrade to v128, I am fine with doing a mandatory upgrade to v3.

Reply:
It would probably still work with v2, but I think there were a few places where changes from v3 were incorporated into v2. That's why I changed it completely.